autoflip_calculators/text_detection_calculator.cc

```diff
@@ -57,6 +57,9 @@ const int EAST_HEIGHT = 320;
 //    options:{
 //      [mediapipe.autoflip.TextDetectionCalculatorOptions.ext]:{
 //        use_visual_scorer: True
+//        model_path: /path/to/modelname.pb
```

Comment:
I think strings need to be in quotes in text proto configs:
model_path: "/path/to/modelname.pb"